test/models/match_test.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class MatchTest < ActiveSupport::TestCase
+  def test_match_must_belong_to_same_organization_as_pet
+    org1 = Organization.create!
+    org2 = Organization.create!
+    pet = Pet.create!(organization: org1,
+      name: "Harry", age: 5, size: "small", breed: "corgi", sex: "m", description: "test pet")
+    match = Match.create(pet: pet, organization_id: org2.id,
+      adopter_account: adopter_accounts(:adopter_account_one))
+    assert !match.valid?
```

Comment:
Feels like there should be something more idiomatic for asserting a falsy value here but I'm not finding it on a quick google search. Oh well! This looks great as-is.

Reply:
I agree and same experience here. I'll merge for now and ask if anyone knows better

Reply:
`refute()` [is the opposite of](https://ruby-doc.org/3.1.2/gems/minitest/Minitest/Assertions.html#method-i-refute) `assert()`, but I'll confess I don't see it used all that often so I can't argue for its idiomacity†.

```suggestion
    refute match.valid?
```

† Which is totally a word I just made up.

Reply:
[Minitest::Expectations](http://docs.seattlerb.org/minitest/Minitest/Expectations.html#method-i-wont_be) are also available.

```suggestion
    _(match).wont_be :valid?
```

Reply:
Sounds like `refute()` is about as idiomatic to Ruby as `idiomacity` is idiomatic to English.

Reply:
I cannot refute that conclusion.